content/en/synthetics/troubleshooting/_index.md

```diff
@@ -87,6 +87,12 @@ To fix it, go edit your recording, open the advanced options of the step that is
 
 {{< img src="synthetics/fix_user_locator.mp4" alt="Fixing User Locator error" video="true"  width="100%" >}}
 
+## API tests
```

Comment:
Can we put API tests before Browser tests? So that it matches the order we have in the TOC as well